src/powerbi/spec/test_data/userLoadedDatasetWithNullRankValues.ts

```diff
@@ -0,0 +1,336 @@
+                                "left": {
+                                    "_kind": 2,
+                                    "source": {
+                                        "_kind": 0,
+                                        "entity": "NullValues"
+                                    },
+                                    "ref": "Name"
+                                },
+                                "right": {
+                                    "_kind": 17,
+                                    "type": {
+                                        "underlyingType": 1,
+                                        "category": <any>null
+                                    },
+                                    "value": "Plane",
+                                    "valueEncoded": "'Plane'"
+                                }
+                            },
+                            "right": {
+                                "_kind": 13,
+                                "comparison": 0,
+                                "left": {
+                                    "_kind": 2,
+                                    "source": {
+                                        "_kind": 0,
+                                        "entity": "NullValues"
+                                    },
+                                    "ref": "b"
+                                },
+                                "right": {
+                                    "_kind": 17,
+                                    "type": {
+                                        "underlyingType": 0,
+                                        "category": <any>null
+                                    },
+                                    "value": <any>null,
+                                    "valueEncoded": "null"
+                                }
+                            }
+                        },
+                        "_key": {}
+                    }
+                ],
+                "identityFields": [
+                    {
+                        "_kind": 2,
+                        "source": {
+                            "_kind": 0,
+                            "entity": "NullValues"
+                        },
+                        "ref": "Name"
+                    },
+                    {
+                        "_kind": 2,
+                        "source": {
+                            "_kind": 0,
+                            "entity": "NullValues"
+                        },
+                        "ref": "b"
+                    }
+                ]
+            }
+        }
+    ]
+};
+
+/* tslint:enable */
+
+import * as _ from "lodash";
+export default function userLoadedDatasetWithNullRankValues() {
+    "use strict";
+    const clonedOptions = <powerbi.VisualUpdateOptions><any>_.cloneDeep(data);
+
+    // Make sure to disable animations
+    _.merge(clonedOptions.dataViews[0].metadata, {
```

Comment:
Why not use rest/spread here instead of lodash? Lodash is adding 500k to your final payload.